src/core/Akka.MultiNodeTestRunner/Program.cs

```diff
@@ -64,13 +71,40 @@ class Program
         ///                  as the test binary.
         ///     </description>
         /// </item>
+        /// <item>
+        ///     <term>-Dmultinode.listen-address={ip}</term>
```

Comment:
New commandline parameters used to specify which IP and port the TCP logger should listen on. Optional - has a set of safe defaults implemented below.